docs/language-usage.rst

```diff
@@ -127,8 +127,8 @@ to content stored in `Google Cloud Storage`_. We can use the
 
   .. code-block:: python
 
-     >>> document = client.document_from_blob(bucket='my-text-bucket',
-     ...                                      blob='sentiment-me.txt')
+     >>> document = client.document_from_blob('my-text-bucket',
+     ...                                      'sentiment-me.txt')
```

Comment:
Got it. I was trying to avoid cross-package dependencies. That's what they do in node (i.e. depend on `storage`)